crates/air-lib/air-parser/src/ast/instructions/traits.rs

```diff
@@ -68,6 +70,12 @@ impl fmt::Display for Ap<'_> {
     }
 }
 
+impl fmt::Display for ApMap<'_> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "ap {} {} {}", self.key, self.value, self.map)
```

Comment:
The map `ap` syntax looks like `(ap (x y) $z)`. I think it worth repeating here.